packages/react-components/src/components/SendBox.test.tsx

```diff
@@ -93,7 +96,7 @@ describe('SendBox should return correct value with a selected mention', () => {
     });
     // Select the suggestion
     await selectFirstMention();
-    expect((input as HTMLInputElement).value).toBe(trigger + suggestions[0].displayText);
+    expect((input as HTMLInputElement).value).toBe(trigger + suggestions[0]?.displayText);

```

Comment:
A bit inconsistency here.
I think the approach of returning early and throwing an error makes more sense.
    if (!suggestions[0]) {
      throw new Error('No suggestions found');
    }

Reply:
The reason I didn't for these was they are inside the expect block, so in this case the developer will get an error like `myString did not match triggerundefined` if this happens to be undefined